<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Selecione uma opção:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mostrar conta(s)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adicionar conta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Editar conta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remover conta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gerar senha aleatória</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adicionar tag</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remover tag</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Padrao"/>
     <widget class="QWidget" name="MostrarContas">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="SelecTagMC">
           <item>
            <property name="text">
             <string>Selecione uma tag:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Banco</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>350</width>
            <height>199</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QWidget" name="WidgetContainer" native="true"/>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AdicionarConta">
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Senha:      </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="SenhaAC"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Descrição:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="DescricaoAC">
         <property name="maximumSize">
          <size>
           <width>122</width>
           <height>91</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tag:          </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="TagAC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Selecione uma opção:</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="BotaoConfirmarAC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Confirmar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Título:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TituloAC"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EditarConta">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Título:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="TituloEC"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Senha:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="SenhaEC"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Descrição:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="DescricaoEC"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Tag:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="BotaoConfirmaEC">
         <property name="text">
          <string>Confirmar edição</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="SelecTituloEC">
         <item>
          <property name="text">
           <string>Selecione um título de uma conta:</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="SelecTagEC">
         <item>
          <property name="text">
           <string>Selecione uma tag:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Banco</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="TagEC"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RemoverConta">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="1" column="0">
        <widget class="QComboBox" name="SelecTagRC">
         <item>
          <property name="text">
           <string>Selecione uma tag:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Banco</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Título:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="TituloRC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Senha:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="SenhaRC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Descrição:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="DescricaoRC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Tag:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="BotaoConfirmaRC">
         <property name="text">
          <string>Confirmar remoção</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="SelecTituloRC">
         <item>
          <property name="text">
           <string>Selecione um título de uma conta:</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="TagRC">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SenhaAleatoria">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Número de caracteres:   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBoxabc">
           <property name="text">
            <string>abc...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxABC">
           <property name="text">
            <string>ABC...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox123">
           <property name="text">
            <string>123...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxExtras">
           <property name="text">
            <string>%#@...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Senha gerada:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SenhaGerada">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AdicionarTag">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Tag's já existentes:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="TagsExistentes"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Nome da nova tag:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="NomeTagNova">
         <property name="maxLength">
          <number>20</number>
         </property>
         <property name="placeholderText">
          <string>Máximo de 20 caracteres</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="BotaoConfirmarAT">
         <property name="text">
          <string>Confirmar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RemoverTag">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="SelecTagRT">
         <item>
          <property name="text">
           <string>Selecione uma tag:</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BotaoConfirmaRT">
         <property name="text">
          <string>Confirmar remoção</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BotaoSair">
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tempo restante da sessão:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tempo">
       <property name="text">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>TituloAC</tabstop>
  <tabstop>SenhaAC</tabstop>
  <tabstop>DescricaoAC</tabstop>
  <tabstop>TagAC</tabstop>
  <tabstop>BotaoConfirmarAC</tabstop>
  <tabstop>TagsExistentes</tabstop>
  <tabstop>NomeTagNova</tabstop>
  <tabstop>BotaoConfirmarAT</tabstop>
  <tabstop>SelecTagMC</tabstop>
  <tabstop>TituloEC</tabstop>
  <tabstop>SenhaEC</tabstop>
  <tabstop>DescricaoEC</tabstop>
  <tabstop>BotaoConfirmaEC</tabstop>
  <tabstop>TituloRC</tabstop>
  <tabstop>SenhaRC</tabstop>
  <tabstop>DescricaoRC</tabstop>
  <tabstop>BotaoConfirmaRC</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>checkBoxabc</tabstop>
  <tabstop>checkBoxABC</tabstop>
  <tabstop>checkBox123</tabstop>
  <tabstop>checkBoxExtras</tabstop>
  <tabstop>SenhaGerada</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>SelecTagRT</tabstop>
  <tabstop>BotaoConfirmaRT</tabstop>
  <tabstop>BotaoSair</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BotaoSair</sender>
   <signal>clicked()</signal>
   <receiver>Menu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
